content/docs/guides/crosswalk/kubernetes/aws/02-kubernetes-stacks/01-cluster-configuration.md

```diff
@@ -0,0 +1,672 @@
+  * Desired capacity, min, and max sizes of the node group.
+
+**General**
+
+  * Kubernetes [labels][k8s-labels] and [taints][k8s-taints] for worker node segmentation and scheduling.
+  * Tags for resources under cluster management.
+
+	```typescript
+	import * as eks from "@pulumi/eks";
+	
+    // Create a Standard node group of t2.medium workers.
+    const ngStandard = new eks.NodeGroup(`${projectName}-ng-standard`, {
+        cluster: cluster,
+        instanceProfile: new aws.iam.InstanceProfile("ng-standard-instanceProfile", {role: stdNodegroupIamRoleName}),
+        nodeAssociatePublicIpAddress: false,
+        nodeSecurityGroup: cluster.nodeSecurityGroup,
+        clusterIngressRule: cluster.eksClusterIngressRule,
+        amiId: "ami-0ca5998dc2c88e64b", // k8s v1.14.7 in us-west-2
+        instanceType: "t2.medium",
+        desiredCapacity: 3,
+        minSize: 3,
+        maxSize: 10,
+        labels: {"amiId": "ami-0ca5998dc2c88e64b"},
+        cloudFormationTags: clusterName.apply(clusterName => ({
+            "CloudFormationGroupTag": "true",
+            "k8s.io/cluster-autoscaler/enabled": "true",
+            [`k8s.io/cluster-autoscaler/${clusterName}`]: "true",
+        })),
+    }, {
+        providers: { kubernetes: cluster.provider},
+    });
+
+    // Create a 2xlarge node group of t3.2xlarge workers with taints for special workloads.
+    const ng2xlarge = new eks.NodeGroup(`${projectName}-ng-2xlarge`, {
+        cluster: cluster,
+        instanceProfile: new aws.iam.InstanceProfile("ng-2xlarge-instanceProfile", {role: perfNodegroupIamRoleName}),
+        nodeAssociatePublicIpAddress: false,
+        nodeSecurityGroup: cluster.nodeSecurityGroup,
+        clusterIngressRule: cluster.eksClusterIngressRule,
+        amiId: "ami-0ca5998dc2c88e64b", // k8s v1.14.7 in us-west-2
+        instanceType: "t3.2xlarge",
+        desiredCapacity: 3,
+        minSize: 3,
+        maxSize: 10,
+        labels: {"amiId": "ami-0ca5998dc2c88e64b"},
+        taints: { "special": { value: "true", effect: "NoSchedule"}},
+        cloudFormationTags: clusterName.apply(clusterName => ({
+            "CloudFormationGroupTag": "true",
+            "k8s.io/cluster-autoscaler/enabled": "true",
+            [`k8s.io/cluster-autoscaler/${clusterName}`]: "true",
+        })),
+    }, {
+        providers: { kubernetes: cluster.provider},
+    });
+	```
+
+## Use the Cluster
+
+After running `pulumi up` the cluster will be created and there will exist
+[Pulumi outputs][pulumi-outputs] with fields like the cluster's `kubeconfig`
+and it's cluster name for reference and usage.
+
+In EKS, the AWS account caller will be placed into the
+`system:masters` Kubernetes RBAC group by default. The `kubeconfig`
+generated will cater to this primary cluster creator use-case, and it must be
+copied, and reconfigured to use with other IAM roles the caller assumes, as
+demonstrated below in [Use Kubernetes RBAC](#use-kubernetes-rbac).
+
+### Access the cluster
```

Comment:
Nit, inconsistent header capitalization. In other guides, like Crosswalk for AWS, we capitalize.

Reply:
Done